webui/react/src/constants/states.ts

```diff
@@ -28,17 +28,24 @@ const jobStates: Array<JobState> = [
   JobState.SCHEDULED,
   JobState.SCHEDULEDBACKFILLED,
 ];
-export const killableRunStates: CompoundRunState[] = [
+const activeStates: Array<RunState> = [
```

Comment:
nit: add a jsdoc style `/* are states that ... */` comment to clarify the potential future usecases

Reply:
✅